ONSdigital/ras-rm-survey
# Add database schema

Creates and automigrates to the database schema specified in the ERD

**How to test**
Deploy to your sandbox, check the logs to ensure there aren't any errors, then port forward to your DB and check you have the surveyv2 schema

![image](https://user-images.githubusercontent.com/6749550/95347289-a298f200-08b4-11eb-87aa-9f8082328ef5.png)
